lib/slim_lint/runner.rb

```diff
@@ -19,9 +19,14 @@ def run(options = {})
 
```

Comment:
Could we skip calling `extract_applicable_files` since it isn't necessary if you're passing in a single file via standard input?